[TEST MERGE] Medbay Maintenance Upgrade (Delta/Kerberos)

ItsMeJaba opened the pull request:
Пройдено:
 * 7+ дней обсуждения с игроками: https://discord.com/channels/617003227182792704/930740675350233098/992088564865183894
 * Закрытое обсуждение с администрацией:
 https://discord.com/channels/617003227182792704/994216391584534581

# Общий план основного изменения:
* Значительное расширение технических тоннелей медицинского блока и дорм
* Создание безопасной возможности эвакуации из шаттла отбытия
![full](https://user-images.githubusercontent.com/91949115/178036855-e470a467-b42c-4664-927e-c51aa1300cad.png)

# Реворк вирусологии
![image](https://user-images.githubusercontent.com/91949115/178037977-6f77b0c0-bda6-4139-ba20-de332f71968b.png)
 Отходы сбрасываются в мусоросжигатель на кольце
 
# Мед.блок
* Расширение морга за счет удаления технического помещения
* В морг добавлены настенные шкафчики для хранения вещей
![image](https://user-images.githubusercontent.com/91949115/179651395-3b319841-b37e-44f0-a393-b3e4cd7fa52f.png)
* Склад разделен на основное снаряжение и склад ВКД
* Палаты уменьшены до 1 двухместной
* Уменьшение кабинета СМО на 1 тайл в ширину
* Выселение психолога в основной коридор
![image](https://user-images.githubusercontent.com/91949115/179651462-d12e6b9a-ff12-4c9a-a348-977c292c7e30.png)

# Единообразие версий карты
На данный момент сложилась ситуация, при которой все более геморнее сводить версии карты, т.к. кроме форс пуша оно адекватно и быстро не решается. Поэтому привел к актуальной карте тестовой ветви, добавив изменения ПРов: #1266 и #1327 - обновления РНД и фикса на это обновление соответственно, а также изменил наномапу под эти изменения. Ввиду этого, можно, не боясь, форспушить этот ПР, т.к. он ничего не снесет.

ItsMeJaba committed:
Кольцо

ItsMeJaba committed:
Наномапа

dikers93 commented:
@ItsMeJaba думаю что на нано мапу повреджения не нужно вносить того тоннеля, по идее то план идеальной станции а не разрухи

dj-34 commented:
> @ItsMeJaba думаю что на нано мапу повреджения не нужно вносить того тоннеля, по идее то план идеальной станции а не разрухи

Наномапа нужна для корректного отображения камер на ней, если там установят камеры, то будет показываться, что они в космосе.

dikers93 commented:
через +- 4 часа будет апдейт, не мержить до него

ItsMeJaba committed:
little bit of fixes

Для единообразия версий переносит изменения обновления РНД и фикс этого РНД. Также обновляет наномапу с этими изменениями
